cypress/plugins/seedEverything.js

```diff
@@ -0,0 +1,63 @@
+const { testUtils } = require('@cumulus/api');
+const serveUtils = require('@cumulus/api/bin/serveUtils');
+const { eraseDataStack } = require('@cumulus/api/bin/serve');
+
+const collections = require('../fixtures/seeds/collectionsFixture.json');
+const executions = require('../fixtures/seeds/executionsFixture.json');
+const granules = require('../fixtures/seeds/granulesFixture.json');
+const providers = require('../fixtures/seeds/providersFixture.json');
+const rules = require('../fixtures/seeds/rulesFixture.json');
+
+// Test values
+const stackName = 'localrun';
```

Comment:
So I don't really like the Cypress code having to be aware of the internals of the local API, including these values or passing around the returned `esClient` and `esIndex`.

It seems possible to me that that we can expose a `resetDataStack` helper that handles fully resetting things without having to pass in any parameters.

I assume that the retrieval of `esClient` and `esIndex` is based on needing to index data after adding it to the DB. But I'm pretty sure we could also refactor those so that passing them in from here is unnecessary

Reply:
I'll make some comments on the API PR

Reply:
Actually I was trying to decouple it some so that this could be used by any person / client who wanted to, not just the tests with these values.  But I can clean it up some.

Reply:
meaning you could use the localAPI with any bucket/index, not just the default ones

Reply:
But the bucket is hardcoded and the index comes back from the helper which is based on the stackName? Is the intent here to be able to use these utilities with your real AWS stack and not just the local API?

Reply:
It was mostly to be flexible with the names.  No intention of running with real AWS.  
But maybe that was overthinking.